centraldogma/data/merged_entry.py

```diff
@@ -0,0 +1,58 @@
+#  Copyright 2022 LINE Corporation
+#
+#  LINE Corporation licenses this file to you under the Apache License,
+#  version 2.0 (the "License"); you may not use this file except in compliance
+#  with the License. You may obtain a copy of the License at:
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations
+#  under the License.
+from typing import Generic, TypeVar, List, Any
+
+from centraldogma import util
+from centraldogma.data.entry import EntryType
+from centraldogma.data.revision import Revision
+from centraldogma.exceptions import CentralDogmaException, EntryNoContentException
+
+T = TypeVar("T")
+
+
+class MergedEntry(Generic[T]):
+    @staticmethod
+    def from_dict(json: Any):
+        paths: List[str] = json["paths"]
+        if len(paths) == 0:
+            raise CentralDogmaException(f"paths is unexpectedly empty for json: {json}")
```

Comment:
If error handling is hesitated, there is an alternative way not to use `try-except` like below.
```suggestion
        paths = json["paths"] if "paths" in json and type(json["paths"]) is list else None
        if not paths:
            raise CentralDogmaException(f"paths is unexpectedly empty for json: {json}")
```

Reply:
https://github.com/line/centraldogma-python/pull/34#discussion_r931858272 <- Addressed here